src/components/scenes/SettingsScene.tsx

```diff
@@ -178,15 +178,14 @@ export class SettingsSceneComponent extends React.Component<Props, State> {
           if (text !== username) return s.strings.delete_account_verification_error
           await this.props.account.deleteRemoteAccount()
           await this.props.logoutRequest()
-          await this.props.context.deleteLocalAccount(username)
-
+          await this.props.context.deleteLocalAccount(username).then(() => {
```

Comment:
Great, but since this function is already `async`, you can ditch the whole `.then(() =>` bit and just show the modal on the next line.